src/client/reducers/uiState/index.js

```diff
@@ -15,72 +15,72 @@ const ui = {
       verified: false,
       fields: []
     },
-    '/personal-information/name-and-general-information': {
+    '/veteran-information/name-and-general-information': {
```

Comment:
Oh I see, we're not changing the path names, just the section names. This is a bit confusing, perhaps we should change both?